### ISSUE TITLE
Unable to record time for an exercise that normally records reps

### ISSUE THREAD

**Issue by quinncomendant** [mazurio/fit-for-reddit-ios#12]
I'm not able to record a time for an exercise that normally records reps, for example wrist exercises. Recording time does work for those that expect to record time (body-line exercises). Example: 

![unable to record time-2](https://cloud.githubusercontent.com/assets/6884276/18256444/1ce22b40-737b-11e6-9eac-9c645cc1b1df.gif)

**Comment by mazurio** [comment]
Thanks for raising.

This is not a bug. There are two types of exercises "weighted" and "timed", all reps/weight based exercises are "weighted" whereas "timed" exercises contain only timer (e.g. Plank).

"weighted" exercises cannot log time whereas "timed" exercises cannot log weight/reps.

This is both a design decision and a feature.

Wrist Mobility Exercise is the only one that's really tricky. It should be timed exercise but it was put as a "weighted" from the beginning.

I will look into whether that can be changed now without messing future graphs/data in the DB.

**Comment by quinncomendant** [comment]
If an exercise cannot be both "timed" and "weighted" then I wonder if it would be good to force the user behavior by allowing only timer _or_ reps counter. Why offer a timer when it's recommended to measure reps instead, and vise versa?

**Comment by mazurio** [comment]
You are right, it shouldn't be there. The only reason why it was there at the beginning was to make the transition from "timer only" to "reps logger" smoother as it was a new feature at the time.

Another reason was that some people use timer as a "rest timer" which is not implemented yet.

For Wrist Mobility I use timer to do the exercise for 2 minutes but then log 5 reps which is bit annoying but works :)

**Comment by quinncomendant** [comment]
It's even questionable to log any of the warmup at all. I'm new to BWF, so maybe I'm wrong (I'm using the app as a learning aid). So far as I understand, warm up isn't progressive and won't increase (e.g., why would you ever do more than 2 min wrist mobility?). I'll probably be following [Antranik's warmup flow](https://youtu.be/8lDC4Ri9zAQ), and log nothing.

**Comment by aaron832** [comment]
I found the timer useful for timing rest periods between strength training sets.  It would be nice to be able to do that again.
